src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/idam/cache/IdamCachedClient.java

```diff
@@ -31,20 +37,28 @@ public IdamCachedClient(
         this.users = users;
         this.accessTokenCache = Caffeine.newBuilder()
             .expireAfter(accessTokenCacheExpiry)
+            .removalListener(this::onCachedIdamTokenRemoval)
+            .build();
+
+        this.userDetailsCache =  Caffeine.newBuilder()
+            .maximumSize(200)
             .build();
     }
 
     public String getAccessToken(String jurisdiction) {
+        log.info("Get access token for jurisdiction: {} ", jurisdiction);
         CachedIdamToken cachedIdamToken = this.accessTokenCache
             .get(jurisdiction, j -> retrieveToken(j));
         return cachedIdamToken.accessToken;
     }
 
     public void removeAccessTokenFromCache(String jurisdiction) {
+        log.info("Remove access token from cache for jurisdiction: {} ", jurisdiction);
         accessTokenCache.invalidate(jurisdiction);
     }
 
     private CachedIdamToken retrieveToken(String jurisdiction) {
+        log.info("Retrieve access token for jurisdiction: {} ", jurisdiction);
```

Comment:
This log entry is very similar to `Get access token ...`. How about adding `... from IdAM` at the end of the message to make it easier to distinguish between these two situations?

Similar things with getting user details.

Reply:
chnaged